### PR TITLE
Upcoming holidays

### PR DESCRIPTION
**Pull request by RiyaSantore04**

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -79,20 +77,41 @@
...
+                    <table class="table table-striped">
+                        <thead>
+                            <tr>
+                                <th scope="col">Holiday Name</th>
+                                <th scope="col">Holiday Type</th>
+                                <th scope="col">Date</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <% for i in 0...@upcoming_holidays.length%>
```
Comment (salvimohan) [inline app/views/dashboard/index.html.erb:92]: why .each

Comment (salvimohan) [inline app/views/dashboard/index.html.erb:92]: @upcoming_holidays blank not handle

Comment (salvimohan) [reply app/views/dashboard/index.html.erb:92]: user partial here

Comment (RiyaSantore04) [reply app/views/dashboard/index.html.erb]: Changes done

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
-# dashboard controller
 class DashboardController < ApplicationController
   before_action :authenticate_employee!
-  skip_before_action :authenticate_employee!, only: [:set_ip]
   def index
     @employees = Employee.where('EXTRACT(DAY FROM date_of_birth) = ?', Date.current.day).where(
       'EXTRACT(MONTH FROM date_of_birth) = ?', Date.current.month
     )
     @recent_join_employees = Employee.where(joining_date: (Time.now.to_date - 2.day)..Time.now.to_date)
+    @upcoming_holidays = Holiday.where(holiday_date: (Time.now.to_date)..Time.now.to_date + 4.day)
```
[the remark this answers is not in the document]
Comment (RiyaSantore04) [reply app/controllers/dashboard_controller.rb]: Changes done

**Pull request merged**